MobileWallet/Screens/Send/AddRecipient/AddRecipientViewController.swift

```diff
@@ -68,8 +68,10 @@ class AddRecipientViewController: UIViewController, UITextFieldDelegate, Contact
 
     let errorMessageView: ErrorView = ErrorView()
 
-    private var selectedRecipientPublicKey: PublicKey? = nil {
+    private var selectedContact: Contact?
+    private var selectedRecipientPublicKey: PublicKey? {
         didSet {
+            if selectedContact != nil { return }
```

Comment:
I think we don't have to display anything in the search box when a contact or non-contact gets selected from the list, the emoji id or alias gets displayed in the next screens already. @Jasonvdb can you confirm please?